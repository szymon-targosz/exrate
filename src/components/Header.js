import React from 'react'

const Header = () => (
   <header className="header">
      <div className="container">
         <h1 className="header__title">Exrate</h1>
      </div>
   </header>
)

export default Header;
