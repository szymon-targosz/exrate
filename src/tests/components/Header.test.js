import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';

describe('COMPONENT - Header', () => {
   test('Should render Header correctly', () => {
      const wrapper = shallow(<Header />);
      expect(wrapper).toMatchSnapshot();
   });
});
