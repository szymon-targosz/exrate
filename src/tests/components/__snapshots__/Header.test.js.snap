// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Header Should render Header correctly 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <h1
      className="header__title"
    >
      Exrate
    </h1>
  </div>
</header>
`;
